src/main/java/org/jabref/gui/entryeditor/EntryEditor.css

```diff
@@ -78,37 +86,54 @@
     -fx-padding: -0.2em 0.1em 0.1em 0.1em;
 }
 
+/*.flatButton .glyph-icon, .flatButtonNoSpaceTop .glyph-icon, .flatButtonNoSpaceBottom .glyph-icon {*/
+    /*-fx-fill: -jr-icon;*/
+/*}*/
+
+/*.flatButton .glyph-icon:hover, .flatButtonNoSpaceTop .glyph-icon:hover, .flatButtonNoSpaceBottom .glyph-icon:hover {*/
+    /*-fx-fill: derive(-jr-icon, -30%);*/
+    /*-fx-effect: dropshadow(one-pass-box, -jr-icon, 5, 0, 1, 1);*/
+/*}*/
+
 .editorPane .glyph-icon {
     -glyph-size: 16px;
 }
 
+
 .tool-bar {
-    -fx-background-color: #dadad8;
+    /*-fx-background-color: -jr-toolbar;*/
     -fx-background-insets: 0;
+    /*-fx-border-color: -jr-gray-1;*/
+    /*-fx-border-width: 1 1 1 1;*/
+    /*-fx-border-radius: 2 2 2 2;*/
+    /*-fx-background-radius: 2 2 2 2;*/
+    /*-fx-text-fill: -jr-icon;*/
+    /*-fx-fill: -jr-icon;*/
 }
 
+
 .tool-bar > .container > .separator {
-    -fx-background-color: #5e5f5e;
+    /*-fx-background-color: -jr-separator;*/
```

Comment:
Reactivate?

Reply:
I would leave it like that. The current separator is derived from the main colors and looks good. `-jr-separator` is not used (or even defined) anymore. It was from my first tests, where I tried to define these myself. When cleaning up, I'm going to remove it.